sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3896,6 +3896,8 @@ _ZN4sycl3_V16mallocEmRKNS0_6deviceERKNS0_7contextENS0_3usm5allocERKNS0_6detail13
 _ZN4sycl3_V16opencl10make_queueERKNS0_7contextEm
 _ZN4sycl3_V16opencl11make_deviceEm
 _ZN4sycl3_V16opencl12make_contextEm
+_ZN4sycl3_V16opencl13has_extensionERKNS0_6deviceERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
```

Comment:
I suppose [`sycl_symbols_windows.dump`](https://github.com/intel/llvm/blob/sycl/sycl/test/abi/sycl_symbols_windows.dump) should be updated as well? It is just we don't have Windows build and tests in our pre-commit and therefore it would only be visible in post-commit that some tests fail